tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -32,6 +32,7 @@ all: serial-targets \
 	qemu-snp-experimental-tarball \
 	shim-v2-tarball \
 	tdvf-tarball \
+	ovmf-tarball \
```

Comment:
I could also build ovmf on my workstation. But as I mentioned in https://github.com/kata-containers/kata-containers/pull/6591#discussion_r1180730266, ovmf and ovmf-sev are overwriting each other:

```
$ tree build/ovmf*
build/ovmf
├── builddir
│   └── edk2-x86_64.tar.gz
└── destdir
    └── opt
        └── kata
            └── share
                └── ovmf
                    └── OVMF.fd
build/ovmf-sev
├── builddir
│   ├── edk2-sev.tar.gz
│   └── sha256sum-kata-static-ovmf-sev.tar.xz
└── destdir
    └── opt
        └── kata
            └── share
                └── ovmf
                    └── OVMF.fd

14 directories, 5 files
```